sites/public/layouts/eligibility.tsx

```diff
@@ -0,0 +1,38 @@
+import Layout from "./application"
+import styles from "./Eligibility.module.scss"
+import { ApplicationTimeout } from "../src/forms/applications/ApplicationTimeout"
+import React, { useContext } from "react"
+import { ProgressNav, t } from "@bloom-housing/ui-components"
+import { EligibilityContext } from "../lib/EligibilityContext"
+import { ELIGIBILITY_SECTIONS } from "../lib/constants"
+import { eligibilityRoute } from "../lib/helpers"
+
+const EligibilityLayout = (props) => {
+  const { eligibilityRequirements } = useContext(EligibilityContext)
+
+  return (
+    <>
+      <ApplicationTimeout />
```

Comment:
I copied this over from the FormsLayout. I think it's valuable to have here, since folks can enter sensitive info in the questionnaire without even being logged in.